lib/analysis/ProjectAnalysis.ts

```diff
@@ -65,6 +66,11 @@ export interface HasAnalysis {
     analysis?: ProjectAnalysis;
 }
 
+/**
+ * All fingerprints from individual scanners, in a record type.
+ */
+export type ConsolidatedFingerprints = Record<string, Fingerprint>;
```

Comment:
Is it okay if this gets quite large?  Our largest projects now have almost 1000 fingerprints.